packages/mobile/package.json

```diff
@@ -229,15 +229,15 @@
       },
       "ios.sim.debug": {
         "binaryPath": "ios/build/Build/Products/Debug-iphonesimulator/celo.app",
-        "build": "set -o pipefail && xcodebuild -workspace ios/celo.xcworkspace -scheme celo-test -configuration Debug -sdk iphonesimulator -derivedDataPath ios/build | bundle exec xcpretty",
+        "build": "set -o pipefail && xcodebuild -workspace ios/celo.xcworkspace -scheme celo-test -configuration Debug -derivedDataPath ios/build | bundle exec xcpretty",
```

Comment:
nit: we can follow up later, but I noticed this change makes it bundle the JS code and assets, so takes more time than it used to.